docs/CalicoSwarm.md

```diff
@@ -0,0 +1,152 @@
+# Running calico-docker networking on a Docker Swarm.
+The following instructions configure a Docker Swarm cluster networked using Calico.  In this tutorial, we will do the following.
+- Configure Docker Swarm on a VM cluster.
+- Configure etcd and Calico on our cluster.
+- Provision workloads on the Swarm, and check they can communicate.
+
+## Prerequisites
+To complete this demo, make sure you have the following prerequisites.
+- Four Linux servers (VMs or bare-metal) with Docker 1.4 or later installed.  The servers should have IP connectivity to each other.
+  - 1 client (to control the Swarm)
```

Comment:
I'm assuming the client, swarm manager and swarm nodes are not pre-requisites and that the instructions below tell you how to install them?